Sources/PrivacyInfo.xcprivacy

```diff
@@ -9,6 +9,15 @@
 	<key>NSPrivacyCollectedDataTypes</key>
 	<array/>
 	<key>NSPrivacyAccessedAPITypes</key>
-	<array/>
+	<array>
+		<dict>
+			<key>NSPrivacyAccessedAPIType</key>
+			<string>NSPrivacyAccessedAPICategoryUserDefaults</string>
+			<key>NSPrivacyAccessedAPITypeReasons</key>
+			<array>
+				<string>CA92.1</string>
```

Comment:
CA92.1 is defined as:
> _"Declare this reason to access user defaults to read and write information that is only accessible to the app itself."_

which doesn't seem right as the cod doesn't access UserDefaults. Is there a more appropreiate value to use, or can the array be empty if none apply?

Reply:
According to apple documentaion we can't keep empty array for below key's , If we are not using in our framework better remove instead of keeping as empty. 

NSPrivacyTrackingDomains
NSPrivacyCollectedDataTypes
NSPrivacyAccessedAPITypes

Reply:
@adzerjitchii @fxwx23 @SethBaldwin @closure11 @tommyming @lpbas @mmackowiak @marcuseckert

Reply:
@ashleymills